### DISCUSSION TITLE
3D пда

### DISCUSSION THREAD

**Discussion by xrLil-Batya** [OGSR/OGSR-Engine#284] [Ideas]
Возможно ли на ТЧ реализовать 3д пда, как в аномалии или в ганслингере?

**Comment by xrSimpAI** [comment]
Если кто-то сделает худовую 3д-модель пда, то можно об этом подумать.

**Comment by OneMorePseudoCoder** [comment]
> Если кто-то сделает худовую 3д-модель пда, то можно об этом подумать.

Можно временно использовать модель из Gunslinger'a или Anomaly.

**Comment by xrSimpAI** [comment]
так в зп ж система рук другая, не подойдет

**Comment by molodoy-chelovek** [comment]
> так в зп ж система рук другая, не подойдет

Есть у кого распакованная геймдата Anomaly?
Скиньте худовую (ну и мировую желательно) модель пда, и .omf файлы с анимациями для пда и рук. 
Если скинут, то на выходных перенесу на ТЧшную систему рук и выложу в этот топик.

**Comment by xrSimpAI** [comment]
Да мне скинули уже.
Всё равно я этим заниматься не буду, как минимум в этом году.

**Comment by I-AmDead** [comment]
> Да мне скинули уже.
> Всё равно я этим заниматься не буду, как минимум в этом году.

Не могли бы поделится моделькой ? А то у меня при экспорте OGF в сдк выдает ошибку duplicate face found

**Comment by xrSimpAI** [comment]
https://cdn.discordapp.com/attachments/775460551727906887/788778661699649606/73c763c76ce2a8ce.rar
